src/views/clinicalGenomic/widgets/sidebar.js

```diff
@@ -431,12 +431,7 @@ function Sidebar() {
         const locations = readerContext?.programs?.map((loc) => loc.location.name);
 
         if (locations) {
-            const selectedNodes = locations.reduce((acc, loc) => {
-                acc[loc] = true;
-                return acc;
-            }, {});
-
-            setSelectedNodes(selectedNodes);
+            setSelectedNodes(locations);
```

Comment:
Figured out why it's misbehaving -- it's expecting an empty object on reset, the code was trying to give it a full object. Using just `locations` would be giving it a list when it's expecting an object, so the below should be preferred:
```suggestion
            setSelectedNodes({});
```

Reply:
Looks like both nodes and cohorts can now be set to empty objects